src/jit/simd.cpp

```diff
@@ -997,12 +997,13 @@ const SIMDIntrinsicInfo* Compiler::getSIMDIntrinsicInfo(CORINFO_CLASS_HANDLE* in
 // Arguments:
 //    type        -  the type of value that the caller expects to be popped off the stack.
 //    expectAddr  -  if true indicates we are expecting type stack entry to be a TYP_BYREF.
+//    structType  -  the class handle to use when normalizing if it is not the same as the stack entry class handle
```

Comment:
I would like for this comment to explain the scenario where this matters - even though I originally understood the context, once I came back to this for review, I couldn't imagine why it wouldn't be the same as the stack entry class handle (even though it's obvious in the context of folding away a static cast!). Anyway, it wouldn't hurt to add something like "(this can happen, e.g., when folding away a static cast, where we want the value popped to have the type that would have been returned)"

Reply:
Fixed.